common/rest_qa_api/base_endpoint.py

```diff
@@ -650,7 +706,7 @@ def __init__(self, base_url):
         self.response_model = response_model
         self.make_url_method = make_url_method
         # Dummy container to prepare request fields
-        self._request = type("Jon_Golt", (object,), dict())()
+        self._request = type("Jon_Galt", (object,), dict())()
```

Comment:
may be type("Dummy_Container", (object,), dict())() ?

Reply:
nope, it's a reference ;)